service/create/load_balancer.go

```diff
@@ -57,6 +57,13 @@ func (s *Service) createLoadBalancer(input LoadBalancerInput) (*awsresources.ELB
 
 	if lbCreated {
 		s.logger.Log("debug", fmt.Sprintf("created ELB '%s'", lb.Name))
+
+		// Assign the ProxyProtocol policy for ingress controller
+		if input.Name == input.Cluster.Spec.Cluster.Kubernetes.IngressController.Domain {
```

Comment:
I don't follow why this check is needed.

Reply:
The proxy protocol policy can only be assigned to the ingress ELB, assigning it to the API ELB gives:
```
$ kubectl get pods
Unable to connect to the server: remote error: tls: record overflow
```